packages/react-core/src/components/Dropdown/InternalDropdownItem.tsx

```diff
@@ -15,7 +16,7 @@ export interface InternalDropdownItemProps extends React.HTMLProps<HTMLAnchorEle
   /** Indicates which component will be used as dropdown item */
   component?: React.ReactNode | string;
   /** Variant of the item. The 'icon' variant should use DropdownItemIcon to wrap contained icons or images. */
-  variant?: 'item' | 'icon';
+  variant?: 'item' | 'icon' | 'routerLink';
```

Comment:
could we make this a boolean instead? e.g. `isRouterLink: boolean` then we can adopt this pattern across components

Reply:
Are you okay with this @kmcfaul, @tlabaj, and @mcoker?

Reply:
Sounds good to me

Reply:
All set!

Reply:
@jschuler @ralpert.  I think this is good now.  Do we need to open follow up issues for other components?

Reply:
@tlabaj yes I think we should track the other components as well so we can align them

Reply:
@jshuler or @rebeccaalpert can you please open an issue for that and listed the impacted components.